custom-ktlint-rules/build.gradle

```diff
@@ -4,16 +4,21 @@ plugins {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(8)
```

Comment:
// somehow megalinter has problem building when the java version is above 11,
        // also I was not able to specify 11 by toolchain, so I used 8